token/metadata.go

```diff
@@ -48,3 +53,178 @@ func (m *Metadata) SpentTokenID() []*token2.ID {
 	}
 	return res
 }
+
+// FilterBy returns a new Metadata containing only the metadata that matches the given enrollment IDs.
+// For Issue actions: the returned metadata will contain only the outputs whose owner has the given enrollment IDs.
```

Comment:
For Issue actions: the returned metadata will contain only the  metadata of the outputs whose owner has the given enrollment IDs.